spread.yaml

```diff
@@ -103,13 +100,15 @@ debug-each: |
 prepare: |
     # this indicates that the server got reused, nothing to setup
     [ "$REUSE_PROJECT" != 1 ] || exit 0
-
     # check that we are not updating
     . "$TESTSLIB/boot.sh"
     if [ "$(bootenv snap_mode)" = "try" ]; then
         echo "Ongoing reboot upgrade process, please try again when finished"
         exit 1
     fi
+    # XXX: hack
```

Comment:
This should be dropped. If tests fail without this they will not represent what actual users will see.